### PR TITLE
WT-11222 Fix run_format_configs to execute failed configs in parallel (v7.0 backport)

### PR DESCRIPTION
**Pull request by svc-bot-sebb**
14fb11463

### CODE REVIEW

**Review by mickgraham-mongodb — CHANGES_REQUESTED** [review]
I believe we need WT-11302 and WT-11299 to make this complete.

**Comment by etienneptl** [comment]
I have pushed the two additional tickets, this should be ready for review @mickgraham-mongodb @raviprakashgiri29

File: test/evergreen.yml [hunk 1]
```diff
@@ -5732,7 +5731,9 @@ buildvariants:
       PATH=/opt/mongodbtoolchain/v4/bin:$PATH
       WT_TOPDIR=$(git rev-parse --show-toplevel)
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
-      LD_LIBRARY_PATH=$WT_BUILDDIR
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
+      LD_PRELOAD=$WT_TOPDIR/TCMALLOC_LIB/lib/libtcmalloc.so
+    CMAKE_PREFIX_PATH: -DCMAKE_PREFIX_PATH="$(pwd)/TCMALLOC_LIB"
```
Comment (mickgraham-mongodb) [inline test/evergreen.yml:5736]: In the order versions we don't run the task on ARM64, this part could be left out but happy either way.

**Review by mickgraham-mongodb — APPROVED** [review]

File: test/evergreen.yml (hunk 1 above)
Comment (etienneptl) [reply test/evergreen.yml]: Sure, removed from the changes!

**Review by mickgraham-mongodb — APPROVED** [review]
LGTM

**Review by raviprakashgiri29 — APPROVED** [review]
Thanks @etienneptl . LGTM

**Pull request merged**